crates/crates_io_trustpub/src/test_keys.rs

```diff
@@ -0,0 +1,70 @@
+//! This module contains a set of RSA keys that must be used only for
+//! testing purposes. The keys are not secure and must not be used
+//! in production!
+
+use jsonwebtoken::errors::Error;
+use jsonwebtoken::{Algorithm, DecodingKey, EncodingKey};
+use serde::Serialize;
+use std::sync::LazyLock;
+
+const PRIVATE_KEY_PEM: &[u8] = br#"
+-----BEGIN PRIVATE KEY-----
```

Comment:
I'm somewhat surprised GitHub allowed me to push this 😂 

As the code comment above says: **DO NOT USE THESE IN PRODUCTION!!!1**